format/spec.md

```diff
@@ -483,6 +485,8 @@ Notes:
 2. For `float` and `double`, the value `-0.0` must precede `+0.0`, as in the IEEE 754 `totalOrder` predicate. NaNs are not permitted as lower or upper bounds.
 3. If sort order ID is missing or unknown, then the order is assumed to be unsorted. Only data files and equality delete files should be written with a non-null order id. [Position deletes](#position-delete-files) are required to be sorted by file and position, not a table order, and should set sort order id to null. Readers must ignore sort order id for position delete files.
 4. The following field ids are reserved on `data_file`: 141.
+5. For `geometry`, this is a point. X = min value of all component points of all geometries in file when edges = PLANAR, westernmost bound of all geometries in file if edges = SPHERICAL. Y = max value of all component points of all geometries in file when edges = PLANAR, northernmost bound of all geometries if edges = SPHERICAL. Z is min value for all component points of all geometries in the file. M is min value of all component points of all geometries in the file. See Appendix D for encoding.
```

Comment:
@jiayuasu @rdblue  does this sound better now?

Can we just say 'westernmost bound of all geometries in the file' for both PLANAR and SPHERICAL

Reply:
I think "edges = PLANAR" should be "edges = LINEAR" and "edges = SPHERICAL" should be "edges = GEODESIC" because "edges" refers to a 1-dimensional line and not a 2-dimensional surface. 

By making these changes, I think it adds clarity and specificity that we are indeed referring to the edges and not implying the type of surface. I think by using the terms "planar" and "spherical" we are implying that the type of surface is either a plane or a sphere. In cases where the spatial model is more complex, like an ellipsoid or geoid, it needs to be very clear to what we are actually referring with the term "edge" and what assumptions are being made about not only the edge but also its related surface. If there is ambiguity about the term, it can cause erroneous assumptions, which can compound spatial computational errors where utmost precision and accuracy are paramount, such as in the defense and intelligence sectors.

Reply:
hm this really doesn't roll off the tongue, could we just live with geodesic?  Nowaday the trend seems to be Geography vs Geometry type actually

Reply:
Personally, I'm good with geodesic; however, there were others that brought up concerns with that term in the GeoParquet group meetup and were more comfortable starting with pseudo-geodesic. I don´t remember the exact details of the issue. Perhaps @paleolimbot, @jiayuasu , and/or @desruisseaux  can chime in with their perspectives on it.

Reply:
> Nowaday the trend seems to be Geography vs Geometry type actually

This trend was initiated by PostGIS, but I don't know how widely it is used outside PostGIS. This is not necessarily a trend that we should follow. The PostGIS's Geography type can handle envelopes crossing the anti-meridian (longitude wraparound), but this is not the only case where wraparounds are needed. Other examples are:

* Radar data using a polar coordinate system.
* Climatological data, where "December" is followed by "January" of no particular year (temporal wraparound).

Instead of a distinct Geography type, a better and more generic solution is to interpret "min" and "max" as "starting point" and "ending point", with interior in the direction of increasing values. So `min > max` (or `west > east`) means that a wraparound happens at infinity:

![image](https://github.com/user-attachments/assets/dc26cd6a-77ce-4747-8346-aa97be48cdee)

This convention is already used by the EPSG geodetic registry, by the GGXF standard, by some versions of the WMS or WCS standards (I do not remember which one exactly). This is the only truly generic approach that I'm aware of. All other approaches work only in special cases.

Reply:
On the "geodesics" versus "pseudo-geodesics" names, I do not have a strong opinion at this time. The rational for "pseudo-generic" is that current implementations are cheating: they do not compute the real geodesic. Instead, they pretend that the ellipsoid is a sphere (e.g., they ignore WGS84 flattening) and apply the formulas for a sphere.

When we pretend that the ellipsoid is a sphere, the results are of course a little bit wrong, but there is some ways to make them "less wrong". For example, some map projections will convert the _geodetic latitudes_ to _authalic latitudes_. An _authalic sphere_ is a sphere with the same surface as the ellipsoid. An _authalic latitude_ is the latitude that a point would have if we deform the ellipsoid until it gets the shape of the authalic sphere. Therefore, for each point in a geometry, the numerical values of the _authalic latitude_ is slightly different than the _geodetic latitude_. If an application converts all geodetic latitudes to authalic latitudes before to perform computations with libraries that assume a sphere, such as S2, then even if the shapes are a little bit wrong, the surfaces are closer to the reality. Conversely, if someone is more interested in shapes rather than surfaces, she/he may use _conformal latitudes_ instead. See for example the list of [auxiliary latitudes on MathWord](https://mathworld.wolfram.com/AuxiliaryLatitude.html).

In summary, even if Iceberg doesn't dive in all this complexity for now, there is a possibility that future versions may want to add something like `AUTHALIC_GEODESIC` (I don't know if putting "authalic geodesic" terms together is orthodox, we would need to verify usages) for geodesics computed on a sphere but with geodetic latitudes converted to authalic latitudes. Idem for other kinds of latitude such as conformal.

The current implementations just ignore all this complexity, use the latitudes _as-is_ and plug the values directly in the spherical formulas without bothering about the fact that this is not quite correct. This is exactly what the "Google Mercator" projection does. EPSG calls that "Popular Visualisation Pseudo Mercator" projection. Hence the proposal for "pseudo" in "pseudo-geodesic", by analogy with "pseudo-Mercator".

Reply:
Thanks for the detailed explanation, will take a further look.  My earlier comment was just that I didnt want to ship something that starts with 'pseudo' as it would brings down the confidence.

I am ok with replacing Planar => Linear and Spherical => Geodesic, if there is general agreement, cc @jiayuasu